translations/TDSv70.js

```diff
@@ -33,7 +33,7 @@ hoot.require('SchemaTools');
 hoot.require('tds70');
 hoot.require('tds70_schema');
 hoot.require('tds70_rules');
-// hoot.require('config');
+hoot.require('config');
```

Comment:
I see other scripts removing this commented out line.  Is it really needed here?  or should it be removed too.

Reply:
Removed alond with some other commented out `config.xxx` lines